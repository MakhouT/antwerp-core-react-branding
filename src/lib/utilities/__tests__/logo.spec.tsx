import * as React from "react";

import * as A from "../..";
import { renderAndMatchSnapshot } from "../../../__tests__/__fixtures__/testUtils";

describe("Logo", () => {
    it("Renders default", () => {
        // Arrange
        const element = <A.Logo
        />;

        // Act and assert
        renderAndMatchSnapshot(element);
    });

    it("Renders absolute", () => {
        // Arrange
        const element = <A.Logo
            absolute
        />;

        // Act and assert
        renderAndMatchSnapshot(element);
    });
});
