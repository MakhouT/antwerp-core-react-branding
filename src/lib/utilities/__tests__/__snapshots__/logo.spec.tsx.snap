// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Renders absolute 1`] = `
<div
  style={
    Object {
      "left": 0,
      "position": "absolute",
      "top": 0,
      "zIndex": 1000,
    }
  }
>
  <img
    alt="Officiëel logo Stad Antwerpen"
    src={Object {}}
  />
</div>
`;

exports[`Logo Renders default 1`] = `
<div
  style={Object {}}
>
  <img
    alt="Officiëel logo Stad Antwerpen"
    src={Object {}}
  />
</div>
`;
